doc/matrix.md

```diff
@@ -57,6 +59,26 @@ $ ni n010p'r 0, a%3, 1' X
 4	3	3
 ```
 
+##1-D Matrix Operations
+Data in row form can be flattened (lengthened?) into a column via `pF_`.
+
+```bash
+$ ni i[a b] i[c d] pF_
+a
+b
+c
+d
+```
+
+Inverting that operation, converting a column to a row with a specified number of fields is done using `Z`, which takes the number of fields as a parameter. 
+
+```bash
+$ ni i[a b] i[c d] pF_ Z2
```

Comment:
nice! i'd also show what happens with a partial line count just so people don't have to wonder about it